tutorials/performance/using_servers.rst

```diff
@@ -119,6 +119,32 @@ This is an example of how to create a sprite from code and move it using the low
         var xform = Transform2D().rotated(deg_to_rad(45)).translated(Vector2(20, 30))
         RenderingServer.canvas_item_set_transform(ci_rid, xform)
 
+ .. code-tab:: csharp
+
+    public partial class MyNode2D : Node2D
+    {
+        // RenderingServer expects references to be kept around. GC will otherwise remove your objects sooner or later...
+        Texture2D texture;
+        // We keep the Rid for later access, for example if we want to remove (clear) the item later.
+        Rid ci_rid;
+
+        public override void _Ready()
+    	{
+    		// Create a canvas item, child of this node.
+    		ci_rid = RenderingServer.CanvasItemCreate();
+    		// Make this node the parent.
+    		RenderingServer.CanvasItemSetParent(ci_rid, GetCanvasItem());
+            // Draw a texture on it.
+            // Remember, keep this reference.
+            Texture2D texture = ResourceLoader.Load<Texture2D>("res://my_texture.png");
```

Comment:
Like the comment says, the texture returned by this method should be stored in the class member declared earlier to keep it around.

Also, the convention in C# projects is to name files using PascalCase.

```suggestion
            _texture = ResourceLoader.Load<Texture2D>("res://MyTexture.png");
```